devtools-ci-index.yaml

```diff
@@ -619,13 +621,93 @@
             ssh_cmd="ssh $sshopts $CICO_hostname"
             $ssh_cmd yum -y install rsync
             rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
-            $ssh_cmd -t "cd payload && /bin/bash cico_build.sh && /bin/bash cico_deploy.sh"
+            $ssh_cmd -t "cd payload && /bin/bash cico_build.sh"
             rtn_code=$?
             cico node done $CICO_ssid
             rsync -e "ssh $sshopts" -Ha --include 'target/**/target/surefire-reports/*.xml' --exclude '*' $CICO_hostname:payload $(pwd)/
             exit $rtn_code
+    publishers:
+      - junit:
+            results: target/**/target/surefire-reports/*.xml
+      - archive:
+            artifacts: che.image.env
+      - trigger:
+          project: eclipse-che-verification
+          threshold: SUCCESS
 
 - job:
+    name: 'eclipse-che-verification'
+    wrappers:
+        - che_credentials_wrapper
+    defaults: global
+    node: devtools
+    builders:
+    builders:
+      - copyartifact:
+          project: devtools-build-run-che-build-master
+          filter: "*che.image.env"
+          optional: false
+          flatten: true
+          do-not-fingerprint: true
+      - shell: |
+            ## At first, deploy Che server
+            . ./che.image.env
+            export VAULT_PORT=8200
+            export VAULT_ROOT_TOKEN_ID=${RHCHEBOT_DOCKER_HUB_PASSWORD}
```

Comment:
I am not sure, I think it should not because I am not setting "set +x". Does the jenkins by default expose variables expansion in exporting a var?